js/components/components.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+/*======================================================================================================
+COMPONENTS
+======================================================================================================*/
+// Change document title
+export const changeDocTitle = (title) => {
+  document.title = `Social Media Page | ${title}`;
+};
+
+// Sign out eventhandler
+export const signOut = () => {
+  document.querySelector("#signOut").addEventListener("click", () => {
+    localStorage.clear();
```

Comment:
```suggestion
    localStorage.remove("token");
```

Clear is too powerful, use remove with a key instead. You may want to keep profile data in storage to make the login process easier

Reply:
Remove does not work. removeItem redirects me to the login, but the local storage is not cleared. I can still access the feed if I write it directly in the url.

Reply:
Added removeItem with the correct key and it now clears from local storage upon signout.